modules/admin/proc-configuring-postgresql-with-tls-using-helm.adoc

```diff
@@ -0,0 +1,146 @@
+[id="proc-configuring-postgresql-with-tls-using-helm_{context}"]
+= Configuring an external PostgreSQL instance with a TLS private key using the Helm Chart
+
+You can edit the Helm Chart default setting to configure an external PostgreSQL database server, such as the Crunchy database. While configuring, you can use a CA certificate, TLS private key, and TLS certificate to secure your database connection using the TLS protocol.
+
+.Prerequisites
+
+* You are using a supported version of PostgreSQL. For more information, see the link:https://access.redhat.com/support/policy/updates/developerhub[Product life cycle page].
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+* You have installed the {product-very-short} application by using the Helm Chart.
+* You have the certificate files and the TLS private key. For more information, refer to your PostgreSQL vendor documentation. 
+
+.Procedure
+
+. Create a certificate secret named `postgress-external-db-cluster-cert`:
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: postgress-external-db-cluster-cert
+type: Opaque
+stringData:
+ postgres-ca.pem: |-
+  -----BEGIN CERTIFICATE-----
+  <ca-certificate-key> <1>
+ postgres-key.key: |-
+  -----BEGIN CERTIFICATE-----
+  <tls-private-key> <2>
+ postgres-crt.pem: |-    
+  -----BEGIN CERTIFICATE-----
+  <tls-certificate-key> <3>
+  # ...
+EOF  
+----
+<1> Provide the CA certificate key.
+<2> Provide the TLS private key.
+<3> Provide the TLS certificate key.
+
+. Create a credential secret named `postgres-cred` to connect with the PostgreSQL instance:
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: postgres-cred
+data: <1>
+  POSTGRES_PASSWORD: <password>
+  POSTGRES_PORT: "<db-port>"
+  POSTGRES_USER: <username>
+  POSTGRES_HOST: <db-host>
+  PGSSLMODE: <ssl-mode> # for TLS connection <2>
+EOF
+----
+<1> Provide credential data to connect with your PostgreSQL instance.
+<2> Provide the value based on the required link:https://www.postgresql.org/docs/15/libpq-connect.html#LIBPQ-CONNECT-SSLMODE[SSL mode].
+
+. Configure your PostgreSQL instance in the Helm configuration file named `values.yaml`:
++
+[source,yaml]
+----
+# ...
+upstream:
+ postgresql:
+   enabled: false  # disable PostgreSQL instance creation <1>
```

Comment:
This updated file is almost perfect. Thank you! Just a few more minor comments:

```suggestion
   enabled: false  # disable PostgreSQL instance creation <1>
   auth:
     existingSecret: postgres-cred   
```
The postgres auth is missing.